conf/nginx.conf

```diff
@@ -13,31 +13,31 @@ client_max_body_size 50M;
 proxy_http_version 1.1;
 proxy_set_header Upgrade $http_upgrade;
 proxy_set_header Connection "upgrade";
-proxy_set_header Host $host;
+proxy_set_header Host $http_host;
 proxy_set_header X-Real-IP $remote_addr;
 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 proxy_set_header X-Forwarded-Proto $scheme;
 
 location / {
-  proxy_pass http://localhost:__PORT__;
+    expires off;
+    more_set_headers "Cache-Control : public, max-age=0, s-maxage=0, must-revalidate";
```

Comment:
Is it relevant like in the model add `always` ? 
```
    more_set_headers "Cache-Control : public, max-age=0, s-maxage=0, must-revalidate" always;
```

Reply:
just changed

Reply:
tested => level4, 
seems that adding always a the end, don't respect the syntax

Reply:
Ok, I rollback to origin...